app/modules/onboarding/assets/js/pages/choose-features.js

```diff
@@ -33,6 +44,16 @@ export default function ChooseFeatures() {
 	if ( 'completed' !== state.steps[ pageId ] ) {
 		skipButton = {
 			text: __( 'Skip', 'elementor' ),
+			action: () => {
+				setAjax( {
+					data: {
+						action: 'elementor_save_onboarding_features',
+						data: JSON.stringify( {
+							features: selectedFeatures,
+						} ),
+					},
+				} );
+			},
```

Comment:
Why do we need to save this if the user chose "Skip"?

Reply:
asked Doron about it

Reply:
Doron confirmed that we are gonna save the data even if user clicks 'skip'